torch/csrc/autograd/VariableTypeManual.cpp

```diff
@@ -375,7 +375,7 @@ static auto registry = torch::RegisterOperators()
     .impl_unboxedOnlyKernel<decltype(VariableType::copy_), &VariableType::copy_>(DispatchKey::VariableTensorId))
   .op(torch::RegisterOperators::options()
     .schema("aten::backward(Tensor self, Tensor? gradient=None, bool keep_graph=False, bool create_graph=False) -> ()")
-    .aliasAnalysis(AliasAnalysisKind::FROM_SCHEMA)
+    .aliasAnalysis(AliasAnalysisKind::CONSERVATIVE)
```

Comment:
Hmmm, I was expecting to see `Tensor(a!) self` in the schema. As given, won't the change to `CONSERVATIVE` change behavior?

Reply:
CONSERVATIVE implies that every argument is mutable and aliased with every other argument. The `(a!)` was actually unnecessary.

Reply:
ah that's still part of the misexport. There is a location that did specify the `(a!)`. I now added its removal to the PR.

Reply:
Can we not use the same string literal for this registration and equivalent (second) one in -[register_prim_ops_fulljit](https://github.com/pytorch/pytorch/pull/36522/files#diff-fb26a0de03180fc8778ce18261715d98R153)? Even now they're not quite the same :)

Reply:
fixed